Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2298,6 +2380,13 @@ void MediaPlayerPrivateGStreamer::createAudioSink()
     gst_element_add_pad(audioSink, gst_ghost_pad_new("sink", pad.get()));
 
     g_object_set(m_playBin.get(), "audio-sink", audioSink, NULL);
+
+    GRefPtr<GstElement> playsink = adoptGRef(gst_bin_get_by_name(GST_BIN(m_playBin.get()), "playsink"));
```

Comment:
Hum I think this would need to be discussed with GStreamer upstream devs. Maybe we could add a flags or some property in playbin to handle this scenario...